Cargo.toml

```
@@ -24,6 +24,7 @@ crc32fast = "1.2"
 anyhow = "1.0"
 clap = { version = "4.0", features = ["derive", "wrap_help"] }
 env_logger = { version = "0.11", default-features = false, features = ["auto-color", "humantime"] }
+rustyline = "16.0.0"
```

```suggestion
rustyline = { version = "16.0.0", default-features = false }
```

If we don't save history (see above), we don't need any of the default features, so adding this should trim the dependency tree a fair bit and reduce binary size.